packit_service/worker/handlers/open_scan_hub.py

```diff
@@ -0,0 +1,125 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+from typing import Tuple, Type
+
+from packit.config import (
+    JobType,
+    aliases,
+)
+from packit_service.worker.checker.abstract import Checker
+from packit_service.worker.events import (
+    OpenScanHubTaskFinishEvent,
+)
+from packit_service.worker.handlers.abstract import (
+    RetriableJobHandler,
+    TaskName,
+    configured_as,
+    reacts_to,
+)
+from packit_service.worker.helpers.build import CoprBuildJobHelper
+from packit_service.worker.helpers.scan import ScanHelper
+
+from packit_service.worker.handlers.mixin import (
+    ConfigFromEventMixin,
+)
+from packit_service.worker.result import TaskResults
+from packit_service.worker.mixin import (
+    LocalProjectMixin,
+    PackitAPIWithUpstreamMixin,
+)
+
+from packit_service.worker.reporting import BaseCommitStatus
+
+logger = logging.getLogger(__name__)
+
+
+@configured_as(job_type=JobType.copr_build)
```

Comment:
I don't even recall what we use this for :eyes:

if it's used for matching the job to the received event, I have a feeling it could break with multiple Copr builds being specified :thinking:

Reply:
Yes exactly, and thus I am checking that in the targets for the copr_build job there is  `fedora-rawhide-x86-64`.